Cell.js

```diff
@@ -0,0 +1,19 @@
+class Cell {
+    constructor(
+        row,
+        column,
+        isMine,
+        numMinesAround,
+        isRevealed,
+        flaggingOn
+    ) {
+        this.column = column;
+        this.row = row;
+        this.isMine = isMine;
+        this.numMinesAround = numMinesAround;
+        this.isRevealed = isRevealed;
+        this.flaggingOn = flaggingOn
```

Comment:
if these values are always constructed with the same value (i.e. `false`) and only updated later, I'd suggest removing them from the constructor params and setting a default value
```suggestion
    constructor(
        row,
        column,
        isMine = false,
        numMinesAround = 0,
        isRevealed = false,
        isFlaggingOn = false
    ) {
        this.column = column;
        this.row = row;
        this.isMine = isMine;
        this.numMinesAround = numMinesAround;
        this.isRevealed = isRevealed;
        this.flaggingOn = flaggingOn
```

you'll have to fact check me on this, but then I'm pretty sure you can do 
```
let cell = new Cell(1, 2);
```
and the values will be defaulted. you can also still include all the params if you want to like you currently have it.

Reply:
I will check this

Reply:
It worked!